src/ui/language_region_notify_popup.ui

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="LanguageRegionNotifyPopup" parent="GtkWindow">
+    <style><class name="settings-popup"/></style>
+    <property name="decorated">false</property>
+    <property name="width-request">380</property>
+    <property name="height-request">420</property>
```

Comment:
too much
![Screenshot from 2024-11-07 09-47-52](https://github.com/user-attachments/assets/ae213bd2-9544-4afd-ae33-85ec84e42a3f)
160 - 200 should be enough